intranet/apps/preferences/tests.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from django.core.urlresolvers import reverse
+from ...test.ion_test import IonTestCase
+
+
+class PreferencesTest(IonTestCase):
+
+    settings_dict = {'showpictures-self': ['on'],
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Duplicate key 'showschedule' in dictionary](https://www.codacy.com/app/tjcsl/ion/file/2380013636/issues/source?bid=3176160&fileBranchId=3221080#l8)